### PR TITLE
First steps of BV in NUE

### PR DESCRIPTION
**Pull request by Bischoff**
+ removing the unused `//xxx_additional_repository` tags

### CODE REVIEW

File: jenkins_pipelines/environments/uyuni-master-qe-build-validation-PRV [hunk 1]
```diff
@@ -22,7 +22,7 @@ node('sumaform-cucumber-provo') {
         parameters([
             string(name: 'cucumber_gitrepo', defaultValue: 'https://github.com/uyuni-project/uyuni.git', description: 'Testsuite Git Repository'),
             string(name: 'cucumber_ref', defaultValue: 'master', description: 'Branch prepared for the MU tested'),
-            string(name: 'tf_file', defaultValue: 'susemanager-ci/terracumber_config/tf_files/Uyuni-Master-build-validation.tf', description: 'Path to the tf file to be used'),
+            string(name: 'tf_file', defaultValue: 'susemanager-ci/terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf', description: 'Path to the tf file to be used'),
```
Comment (nodeg) [inline jenkins_pipelines/environments/uyuni-master-qe-build-validation-PRV:25]: The title states BV in NUE but the new file name is appended with PRV. This seems wrong to me.

**Review by nodeg — CHANGES_REQUESTED** [review]

**Comment by srbarrios** [comment]
I'm fine with it.
But keep in mind that will "break" a feature in terraform.

Currently, in terraform, you can specify custom repositories to be injected at deployment phase at any module, including all these clients.

File: jenkins_pipelines/environments/uyuni-master-qe-build-validation-PRV (hunk 1 above)
Comment (Bischoff) [reply jenkins_pipelines/environments/uyuni-master-qe-build-validation-PRV:25]: I renamed the file to make room for the -NUE one.

Comment (nodeg) [reply jenkins_pipelines/environments/uyuni-master-qe-build-validation-PRV:25]: I see. Thank you for the clarification.

**Pull request merged**